src/web/components/SlotBodyEnd/ReaderRevenueEpic.tsx

```diff
@@ -142,132 +209,39 @@ const MemoisedInner = ({
 
-		const forcedVariant = getForcedVariant('epic');
-		const queryString = forcedVariant ? `?force=${forcedVariant}` : '';
-
-		buildPayload({
-			isSignedIn,
-			countryCode,
-			contentType,
-			sectionName,
-			shouldHideReaderRevenue,
-			isMinuteArticle,
-			isPaidContent,
-			tags,
-			contributionsServiceUrl,
-			isSensitive,
-		})
-			.then((contributionsPayload) =>
-				getBodyEnd(
-					contributionsPayload,
-					`${contributionsServiceUrl}/epic${queryString}`,
-				),
-			)
-			.then((response) => {
-				dataPerf.end();
-				return checkForErrors(response);
+		window
+			.guardianPolyfilledImport(module.url)
+			.then((epicModule: { ContributionsEpic: React.FC<EpicProps> }) => {
+				modulePerf.end();
+				setEpic(() => epicModule.ContributionsEpic); // useState requires functions to be wrapped
+				sendOphanComponentEvent('INSERT', meta);
 			})
-			.then((response) => response.json())
-			.then(
-				(json: {
-					data: { meta: any; module: { url: string; props: any } };
-				}) => {
-					if (!json.data) {
-						return;
-					}
-
-					const { meta, module } = json.data;
-
-					const modulePerf = initPerf('contributions-epic-module');
-					modulePerf.start();
-
-					// eslint-disable-next-line no-restricted-globals
-					window
-						.guardianPolyfilledImport(module.url)
-						.then(
-							(epicModule: {
-								ContributionsEpic: JSX.Element;
-							}) => {
-								modulePerf.end();
-								setEpicMeta(meta);
-								setEpicProps({
-									...module.props,
-									onReminderOpen: sendOphanReminderOpenEvent,
-								});
-								setEpic(() => epicModule.ContributionsEpic); // useState requires functions to be wrapped
-								sendOphanComponentEvent('INSERT', meta);
-							},
-						)
-						// eslint-disable-next-line no-console
-						.catch((error) =>
-							console.log(`epic - error is: ${error}`),
-						);
-				},
-			);
+			// eslint-disable-next-line no-console
+			.catch((error) => console.log(`epic - error is: ${error}`));
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
-	// Should only run once
 	useEffect(() => {
-		if (hasBeenSeen && epicMeta) {
-			const { abTestName } = epicMeta;
+		if (hasBeenSeen && meta) {
+			// Should only run once
+			const { abTestName } = meta;
 
 			logView(abTestName);
 
-			sendOphanComponentEvent('VIEW', epicMeta);
+			sendOphanComponentEvent('VIEW', meta);
 		}
-	}, [hasBeenSeen, epicMeta]);
-
+	}, [hasBeenSeen, meta]);
 	if (Epic) {
 		return (
 			<div ref={setNode} className={wrapperMargins}>
 				{/* eslint-disable-next-line react/jsx-props-no-spreading */}
-				<Epic {...epicProps} />
+				<Epic {...module.props} />
 			</div>
 		);
 	}
 
 	return null;
 };
-
-export const SlotBodyEnd = ({
-	isSignedIn,
-	countryCode,
-	contentType,
-	sectionName,
-	shouldHideReaderRevenue,
-	isMinuteArticle,
-	isPaidContent,
-	isSensitive,
-	tags,
-	contributionsServiceUrl,
-}: Props) => {
-	if (isSignedIn === undefined || countryCode === undefined) {
```

Comment:
Just a note for future readers because it's not immediately clear where this check lives in the new code: waiting for `isSignedIn` and `countryCode` to be defined before calling the support-dotcom-components service is now handled by the `useOnce` in `SlotBodyEnd`.